base/multimedia.jl

```diff
@@ -225,6 +225,18 @@ function displayable(m::MIME)
     return false
 end
 
+
+macro display(exs...)
+    blk = Expr(:block)
+    for ex in exs
+        push!(blk.args, :(print($(sprint(show_unquoted,ex)*" = "))))
+        push!(blk.args, :(display(begin value=$(esc(ex)) end)))
+        push!(blk.args, :(println()))
+    end
```

Comment:
No.  And there may be one display (e.g. a separate window) for one type of object (e.g. an image) and another type of display (e.g. the REPL) for text.

As I result, I'm not quite sure what you are trying to accomplish here that isn't done by `@show`.